docs/tutorials/getting-started/interacting-with-NFTs.md

```diff
@@ -0,0 +1,348 @@
+# Interacting with the NFT Contract using the Rust Casper Client
+
+This document describes interacting with NFTs on a Casper network using the Rust command-line client.
+
+
+## Prerequisites
+
+- Install the contract using the [Quickstart](./quickstart-guide.md) or the [Full Installation](./full-installation-tutorial.md) tutorials
+- Learn to [Query NFT Contracts](./querying-NFTs.md) and save the various hashes and URefs required throughout this document
+
+
+## Table of Contents
+
+1. [Directly Invoking Entrypoints](#directly-invoking-entrypoints)
+
+2. [Minting NFTs](#minting-nfts)
+
+3. [Transferring NFTs](#transferring-nfts)
+
+4. [Checking Balances](#checking-balances)
+
+5. [Approving an Account](#approving-an-account)
+
+6. [Burning NFTs](#burning-nfts)
+
+7.  [Next Steps](#next-steps)
+
+
+## Directly Invoking Entrypoints
+
+With the release of CEP-78 version 1.1, users interacting with a CEP-78 contract that does not use `ReverseLookupMode` should opt out of using the client Wasm files provided as part of the release. Opting out in this situation is recommended, as directly invoking the entrypoints incurs a lower gas cost than using the provided client Wasm to invoke the entrypoint.
+
+You may invoke the `mint`, `transfer`, or `burn` entrypoints directly through either the contract package hash or the contract hash directly.
+
+In the case of `mint`, fewer runtime arguments must be provided, thereby reducing the total gas cost of minting an NFT.
+
+<details>
+<summary><b>Example `mint` using the stored package hash</b></summary>
+
+```bash
+casper-client put-deploy --node-address http://localhost:11101/rpc/ \
+--chain-name "casper-net-1” \
+--payment-amount 5000000000 \
+--secret-key ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem \
+--session-package-hash hash-2b61207cd0e94ce1b1d40801b0abb1ab55fd7dae94c9dcf670292243f3791a30 \
+--session-entry-point "mint" \
+--session-arg "token_owner:key='account-hash-e70dbca48c2d31bc2d754e51860ceaa8a1a49dc627b20320b0ecee1b6d9ce655'" \
+--session-arg "token_meta_data:string='{\"name\": \"John Doe\",\"token_uri\": \"https:\/\/www.barfoo.com\",\"checksum\": \"940bffb3f2bba35f84313aa26da09ece3ad47045c6a1292c2bbd2df4ab1a55fb\"}'"
```

Comment:
In identifier mode hash, you can also pass 'token_hash' as argument to set your custom identifier here, I see you chose identifier mode ordinal but this feature is very usefull to retrieve meta_data from a nft because you alreay have the identifier for the query to retrieve metadata.

Reply:
Good point. I've added a note with commit [4b55cb6](https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/269/commits/4b55cb651703c2e111cb5c0bed7e8f4dc9b8fd52).